src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java

```diff
@@ -43,14 +44,18 @@ public void onMessageDelete(MessageDeleteEvent event) {
     }
 
     private Predicate<Message> isOurResponseTo(long messageId) {
-        String tracker = "re: " + messageId;
+        String tracker = longToBinaryBlankString(messageId);
         return message -> !message.getEmbeds().isEmpty()
                 && message.getAuthor().isYourself()
                 && message.getEmbeds().get(0).getFooter()
                 .flatMap(EmbedFooter::getText)
-                .map(tracker::equals)
-                .orElse(false);
+                .filter(text -> text.startsWith(tracker))
+                .isPresent();
     }
 
-
+    private static String longToBinaryBlankString(long l)  {
+        return Long.toBinaryString(l)
+                .replace('0', '\u200B')
+                .replace('1', '\u200D');
+    }
```

Comment:
What do you think about terminating the binary blank string to not later match on prefixes as you changed to `startsWith`?
Maybe another zero-width characer. Maybe `\u200C`?

Reply:
Or maybe then B for 0, C for 1 and D as terminator

Reply:
Any reason to not have B, C, D in order? :-)

Reply:
I just added the part with the C :-)